plugin/build.gradle

```diff
@@ -12,6 +12,10 @@ buildscript {
     }
 }
 
+plugins {
+    id 'com.github.sherter.google-java-format' version '0.8'
```

Comment:
Latest version is 0.9, released 23 days ago. That version don't work well with our current TravisCI pipeline, so I decided to go with version 0.8.

Reply:
Sounds good, thanks for calling this out.